Adding hrim_sensor_forcetorque_msgs

ibaiape opened the pull request:
Following same message structure as the Force sensor.

ahcorde commented:
why is this repeat it?

```
uint8 MX=0
uint8 MY=1
uint8 MZ=2
```
and 
```
uint8 FX=0
uint8 FY=1
uint8 FZ=2
```

ibaiape commented:
Initially done with the focus of facilitating usage (not using definitions from a different file than the one used; not having to reference SpecsForceAxis when working with ForceAxis).

Modified naming for comprehension, removed duplicate declaration.
